### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by DranxX** [Nurutomo/wabot-aq#930] [Q&A]
Cara ganti nomor owner dimana

**Comment by unx21** [comment]
> Cara ganti nomor owner dimana

Config.js, tapi gausah di jadiin pull request disini

**Reply by DranxX** [reply]
Udah diganti tpi tetep pas .owner munculnya nurutomo

**Comment by unx21** [comment]
> Udah diganti tpi tetep pas .owner munculnya nurutomo

Creator.js
Tambahin nomor ama nick kamu
